tracer/test/test-applications/aspnet/Samples.AspNetMvc5/Handlers/CustomTracingExceptionHandler.cs

```diff
@@ -1,20 +1,28 @@
 using Datadog.Trace;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Web;
 using System.Web.Http.ExceptionHandling;
 
 namespace Samples.AspNetMvc5.Handlers
 {
     public class CustomTracingExceptionHandler : ExceptionHandler
     {
-        public override async Task HandleAsync(ExceptionHandlerContext context, CancellationToken cancellationToken)
+        public override void Handle(ExceptionHandlerContext context)
         {
+#if !OWIN
+            if (HttpContext.Current.Items["ErrorStatusCode"] is int statusCode)
+            {
+                string path = $"~/api/statuscode/{statusCode}";
+                HttpContext.Current.Server.TransferRequest(path, false, "GET", null);
+                return;
+            }
+#endif
+
             using (var scope = Tracer.Instance.StartActive("CustomTracingExceptionHandler.handle-async"))
             {
                 // Set span kind of span to server to pass through server span filtering
                 scope.Span.SetTag(Tags.SpanKind, SpanKinds.Server);
-
-                await base.HandleAsync(context, cancellationToken);
```

Comment:
out of interest, why are we no longer calling this? 🤔 (Maybe just not necessary, can't remember why it was there)

Reply:
The `ExceptionHandler` class allows you to override either `Handle` or `HandleAsync`. Since I wanted to invoke `TransferRequest` and it's synchronous, I decided to switch to overriding `Handle`. As a result, it didn't make any sense to call `base.HandleAsync` anymore